### ISSUE TITLE
Library View, Double Click to Play Playlist

### ISSUE THREAD

**Issue by Nexuist** [ShyykoSerhiy/vscode-spotify#84]
Hello,

In the Spotify app, I can double click on a playlist name to start playing that playlist in shuffle mode. I would like to suggest implementing that functionality in this extension as well.

Also, I would like to see my song library from this extension because not all of my songs are sorted into playlists yet.

If you would like help doing this, let me know. I personally don't think these are really high priority, just nice to haves.

Thanks again for building this extension. It's super helpful and hella customizable.

**Comment by ShyykoSerhiy** [comment]
Is this the "my song library" you are talking about? 
<img width="184" alt="screen shot 2019-01-23 at 10 44 28 am" src="https://user-images.githubusercontent.com/1106995/51618443-577e5600-1efc-11e9-82d1-33fca6df5467.png">
Do you think it's ok to put it inside 'playlists' section of vscode-spotify in the top most position here?:
<img width="348" alt="screen shot 2019-01-23 at 10 49 54 am" src="https://user-images.githubusercontent.com/1106995/51618742-d2477100-1efc-11e9-9220-73926b387f9c.png">

**Comment by Nexuist** [comment]
Yes, that's the right tab I'm talking about. I think it would make sense to keep it as the topmost playlist.

Thanks for the super quick reply! :)

**Comment by GeorgeNance** [comment]
I agree, it feels natural to double click a playlist and for it to start the first song.

**Comment by ShyykoSerhiy** [comment]
Connected issue: https://github.com/spotify/web-api/issues/1297

**Comment by Hug0Albert0** [comment]
Still not resolved, the only playlist I get its the radio ones, not my liked songs
